src/app/header/header.component.html

```diff
@@ -0,0 +1,12 @@
+<header>
+  <img
+    *ngIf="chuckFace"
+    src="assets/images/chuck.png"
+    alt="Chuck Norris with black hat and black sunglasses"
+    class="chuck" />
+  <img
+    *ngIf="!chuckFace"
+    src="assets/images/face.jpeg"
+    alt="Solid face icon"
+    class="face" />
```

Comment:
in this case simplify:
```
  <img
    [src]="chuckFace ? 'assets/images/chuck.png' : 'assets/images/chuck.png'"
    [alt]="chuckFace ? 'Chuck Norris with black hat and black sunglasses' : 'Solid face icon'"
    [ngClass]="chuck" />
```

but consider create new file to store all images with keys, ex:

```
const ASSETS = {
 CHUCK_FACE: 'assets/images/chuck.png'
}
```
you would be able to use this once in whole app and easily change path in the future.

to use in the template:

```
get imageUrl() {
  return ASSETS
}
```

in the template 
`[src]="imageUrl.CHUCK_FACE"`